src/app/models/evt-models.ts

```diff
@@ -15,3 +15,50 @@ export interface PageData {
     originalContent: OriginalEncodingNodeType[];
     parsedContent: GenericElementData[];
 }
+
+export type AttributesData = Array<{ key: string; value: string }>;
+
+export interface OriginalEncodingData {
+    originalEncoding: OriginalEncodingNodeType;
+}
+
+export type NamedEntityType = 'person' | 'place' | 'org' | 'relation' | 'event' | 'generic';
+export interface NamedEntitiesList extends GenericElementData {
+    id: string;
+    label: GenericElementData | string;
```

Comment:
Weird that a label isn't always a string. Check it.

Reply:
If the list element has the child node `<head>` I parse this and set as the label. Otherwise I use the `@type` attribute (that is a `string`). This is because I wanted to save possible formatting within `<head>` element. However, I've checked again the TEI guidelines and all the example of `<head>` within a list only contain text. Thus, we can probably switch to a more simple `string` type for label!